include/deal.II/lac/sparse_decomposition.h

```diff
@@ -237,24 +237,6 @@ class SparseLUDecomposition : protected SparseMatrix<number>,
                    const AdditionalData parameters);
 
   /**
-   * This method is deprecated, and left for backward compability. It will be
-   * removed in later versions.
-   *
-   * @deprecated
-   */
-  template <typename somenumber>
-  void decompose (const SparseMatrix<somenumber> &matrix,
-                  const double                    strengthen_diagonal=0.) DEAL_II_DEPRECATED;
-
-  /**
-   * This method is deprecated, and left for backward compability. It will be
-   * removed in later versions.
-   *
-   * @deprecated
-   */
-  virtual bool is_decomposed () const DEAL_II_DEPRECATED;
-
-  /**
```

Comment:
Good catch. It turns out that this flag is now unused altogether (only ever written to, never read), so I'm going to remove it. But there is also a big blob of documentation in `SparseLUDecomposition` that I will have to review.